hackathon/views.py

```diff
@@ -1,10 +1,61 @@
 from django.views.generic import ListView
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from datetime import datetime
 
-from .models import Hackathon
+from .models import Hackathon, HackAwardCategory, HackTeam, HackProject, HackProjectScore, HackProjectScoreCategory
```

Comment:
Please only import models you need

Reply:
No idea why I thought to import them all! I think it might have been to save time for anyone else who would be using them in the same file. I've removed the unused models now.